src/alr/alr-commands.adb

```diff
@@ -516,7 +207,7 @@ package body Alr.Commands is
    procedure Reportaise_Wrong_Arguments (Message : String) is
    begin
       Alire.Errors.Pretty_Print (Message);
-      raise Wrong_Command_Arguments with Alire.Errors.Set (Message);
+      raise SubCmd.Wrong_Command_Arguments with Alire.Errors.Set (Message);
```

Comment:
I find this mix of `CamelCase` and `Camel_Case` quite jarring... I'd prefer to stick to the latter style.